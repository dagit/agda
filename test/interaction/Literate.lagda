\begin{code}
module Literate where

postulate A : Set
\end{code}

\begin{code}
postulate a : A
\end{code}
