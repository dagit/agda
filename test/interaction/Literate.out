Prelude> Agda2> (agda2-status-action "")
(agda2-info-action "*Type-checking*" "" nil)
(agda2-highlight-clear)
(agda2-info-action "*Type-checking*" "Checking Literate (Literate.lagda).\n" t)
(agda2-info-action "*Type-checking*" "Finished Literate.\n" t)
(agda2-status-action "Checked")
(agda2-info-action "*All Goals*" "" nil)
((last . 1) . (agda2-goals-action '()))
Agda2> 